src/app/_components/most-popular-section/tabs.tsx

```
@@ -0,0 +1,39 @@
+          <TabsTrigger value="account">Wedding</TabsTrigger>
+          <TabsTrigger value="password">Birthday</TabsTrigger>
+        </TabsList>
+        <TabsContent value="account">
+          <Card>
+            <CardHeader>
+              <CardTitle>Wedding</CardTitle>
+              <CardDescription>
+                weddingggggg products...
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="grid gap-6">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)